ios/Classes/CameraPlugin.m

```diff
@@ -929,7 +925,9 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call result:(FlutterResult)re
                                    stringWithFormat:@"flutter.io/cameraPlugin/cameraEvents%lld",
                                                     textureId]
                binaryMessenger:_messenger];
-      [eventChannel setStreamHandler:cam];
+      dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Kaip suprantu čia UI thread'e paleidžia šitą kodą? Gali paaiškinti kam tai reikalinga?

Reply:
Čia kodas iš PR: https://github.com/Lightsnap/flutter_better_camera/pull/50/files 
Lokaliai paleidžiant crashina, kadangi ne tame threade runina.